src/Utility/ThreadGroups.h

```diff
@@ -81,6 +81,10 @@ class ThreadGroups
   //! \brief Indexing operator.
   IntMat& operator[](int i) { return tg[i]; }
 
+  //! \brief Filter current threadgroups through a white list of elements.
+  //! \param[in] elmList The white list of elements
+  ThreadGroups filter(const IntVec& elmList);
```

Comment:
This method should be const, does not touch its members

Reply:
done.